linter/allowlists/default.txt

```diff
@@ -85,6 +87,7 @@ Fred
 Freunde
 Front end
 front end
+Fußball
```

Comment:
Did you get an error message from the linter if this keyword is not in the allowlist? I wonder why it is necessary to include the word "Fußball" here.

Reply:
Yes my the linter complained about Fußball!

Reply:
But not for the German version, presumably? The word should only be added to the English allow list.